hazelcast/src/main/java/com/hazelcast/nio/tcp/handlermigration/IOBalancer.java

```diff
@@ -77,7 +75,7 @@ public IOBalancer(InSelectorImpl[] inSelectors, OutSelectorImpl[] outSelectors,
         this.inLoadTracker = new LoadTracker(inSelectors, loggingService);
         this.outLoadTracker = new LoadTracker(outSelectors, loggingService);
 
-        this.isActive = shouldStart(inSelectors, outSelectors);
+        this.shouldStart = shouldStart(inSelectors, outSelectors);
```

Comment:
me too, but it's misleading. 
it's set to `true` in a constructor even when it's not active yet. it's active only after calling the `start()` method.